locksmith/src/routes/v2/verifier.ts

```diff
@@ -7,18 +7,22 @@ const router = express.Router({ mergeParams: true })
 
 const verifierController = new VerifierController()
 
-router.all('/', authenticatedMiddleware, lockManagerMiddleware)
+router.use('/', authenticatedMiddleware)
```

Comment:
It looks like all routes use `lockManagerMiddleware`. Why not keeping it there too?

Reply:
hm, that's odd. I will check.